\doxysection{Référence de la classe pip.\+\_\+vendor.\+urllib3.\+exceptions.\+Response\+Error}
\hypertarget{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1ResponseError}{}\label{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1ResponseError}\index{pip.\_vendor.urllib3.exceptions.ResponseError@{pip.\_vendor.urllib3.exceptions.ResponseError}}


Graphe d\textquotesingle{}héritage de pip.\+\_\+vendor.\+urllib3.\+exceptions.\+Response\+Error\+:
% FIG 0


Graphe de collaboration de pip.\+\_\+vendor.\+urllib3.\+exceptions.\+Response\+Error\+:
% FIG 1
\doxysubsubsection*{Attributs publics statiques}
\begin{DoxyCompactItemize}
\item 
str \mbox{\hyperlink{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1ResponseError_a07812ee5d5e8481d968a2de35dcadb4e}{GENERIC\+\_\+\+ERROR}} = "{}too many error responses"{}
\item 
str \mbox{\hyperlink{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1ResponseError_ac10aa8fa27ec217e5aaf8fe8ddd83bd8}{SPECIFIC\+\_\+\+ERROR}} = "{}too many \{status\+\_\+code\} error responses"{}
\end{DoxyCompactItemize}


\doxysubsection{Description détaillée}
\begin{DoxyVerb}Used as a container for an error reason supplied in a MaxRetryError.\end{DoxyVerb}
 

\doxysubsection{Documentation des données membres}
\Hypertarget{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1ResponseError_a07812ee5d5e8481d968a2de35dcadb4e}\label{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1ResponseError_a07812ee5d5e8481d968a2de35dcadb4e} 
\index{pip.\_vendor.urllib3.exceptions.ResponseError@{pip.\_vendor.urllib3.exceptions.ResponseError}!GENERIC\_ERROR@{GENERIC\_ERROR}}
\index{GENERIC\_ERROR@{GENERIC\_ERROR}!pip.\_vendor.urllib3.exceptions.ResponseError@{pip.\_vendor.urllib3.exceptions.ResponseError}}
\doxysubsubsection{\texorpdfstring{GENERIC\_ERROR}{GENERIC\_ERROR}}
{\footnotesize\ttfamily str pip.\+\_\+vendor.\+urllib3.\+exceptions.\+Response\+Error.\+GENERIC\+\_\+\+ERROR = "{}too many error responses"{}\hspace{0.3cm}{\ttfamily [static]}}

\Hypertarget{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1ResponseError_ac10aa8fa27ec217e5aaf8fe8ddd83bd8}\label{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1ResponseError_ac10aa8fa27ec217e5aaf8fe8ddd83bd8} 
\index{pip.\_vendor.urllib3.exceptions.ResponseError@{pip.\_vendor.urllib3.exceptions.ResponseError}!SPECIFIC\_ERROR@{SPECIFIC\_ERROR}}
\index{SPECIFIC\_ERROR@{SPECIFIC\_ERROR}!pip.\_vendor.urllib3.exceptions.ResponseError@{pip.\_vendor.urllib3.exceptions.ResponseError}}
\doxysubsubsection{\texorpdfstring{SPECIFIC\_ERROR}{SPECIFIC\_ERROR}}
{\footnotesize\ttfamily str pip.\+\_\+vendor.\+urllib3.\+exceptions.\+Response\+Error.\+SPECIFIC\+\_\+\+ERROR = "{}too many \{status\+\_\+code\} error responses"{}\hspace{0.3cm}{\ttfamily [static]}}



La documentation de cette classe a été générée à partir du fichier suivant \+:\begin{DoxyCompactItemize}
\item 
env/lib/python3.\+12/site-\/packages/pip/\+\_\+vendor/urllib3/\mbox{\hyperlink{pip_2__vendor_2urllib3_2exceptions_8py}{exceptions.\+py}}\end{DoxyCompactItemize}
